contracts/src/gateway/router/TopDownFinalityFacet.sol

```diff
@@ -35,6 +35,15 @@ contract TopDownFinalityFacet is GatewayActorModifiers {
         s.validatorsTracker.batchStoreChange(changeRequests);
     }
 
+    /// @notice Get the configuration numbers in the validatorTracker struct that shadows the configuration changes
```

Comment:
Nit: the docs leak internal structure and don't actually state/explain the return values